src/test/features/dashboard/routes/claimStatus/full-defence.ts

```diff
@@ -74,15 +73,15 @@ const testData = [
       ...directionsQuestionnaireDeadline
     },
     claimantAssertions: [
-      'You’ve rejected the defendant’s admission',
-      `They said they owe ${NumberFormatter.formatMoney(defenceWithAmountClaimedAlreadyPaidData.paymentDeclaration.paidAmount)}`,
-      'complete a directions questionnaire',
+      'Wait for the court to review the case',
```

Comment:
question: do we have any tests for the new content ?

Reply:
Added tests for mediation agreed case